controller/testController.go

```
@@ -58,10 +68,62 @@ func (c *TestController) test(ctx *gin.Context) {
+// @Description do a delete on item uui
+// @Tags test
+// @Accept json
+// @Produce json
+// @Success 200
+// @Param        uuid   path      string  true  "Test model UUID"
+// @Router /test/{uuid} [delete]
+func (c *TestController) delete(ctx *gin.Context) {
+	id, err := uuid.Parse(ctx.Param("uuid"))
+	if err != nil {
+		zap.S().Errorf("error parsing uuid value = %s", ctx.Param("uuid"))
+		ctx.AbortWithError(http.StatusBadRequest, err)
```

golangci-lint (1.64.8)

Error return value of `ctx.AbortWithError` is not checked

(errcheck)